src/components/Register/index.tsx

```diff
@@ -1,27 +1,32 @@
-import { zodResolver } from '@hookform/resolvers/zod'
-import { useForm } from 'react-hook-form'
-import { toast } from 'sonner'
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useForm } from "react-hook-form";
+import { toast } from "sonner";
 
-import { RegisterForm, RegisterFormSchema } from '../zod.schema'
-import { Participants } from './components/Participants'
-import { TermsAndConditions } from './components/TermsAndConditions'
-import { ProjectForm } from './components/ProjectForm'
+import { RegisterForm, RegisterFormSchema } from "../zod.schema";
+import { Participants } from "./components/Participants";
+import { TermsAndConditions } from "./components/TermsAndConditions";
+import { ProjectForm } from "./components/ProjectForm";
+import { createClient } from "@supabase/supabase-js";
 
 const DEFAULT_REGISTER_FORM_VALUES: RegisterForm = {
-  project_name: '',
-  project_description: '',
-  project_url: '',
+  project_name: "",
+  project_description: "",
+  project_url: "",
   participants: [
     {
-      participant_name: '',
-      participant_country: '',
-      participant_email: '',
+      participant_name: "",
+      participant_country: "",
+      participant_email: "",
     },
   ],
   terms_and_conditions: false,
-} as const
+} as const;
 
 export const Register = () => {
+  const supabaseUrl = import.meta.env.VITE_SUPABASE_URL;
+  const supabaseKey = import.meta.env.VITE_SUPABASE_KEY;
+  const supabase = createClient(supabaseUrl, supabaseKey);
```

Comment:
You should extract this outside of the component so it doesn't create a new client on every render.

Reply:
I can't merge your PR until you resolve errors in deploy

Reply:
@MarcosNASA React framework issue